tests/refer-and-monitor-and-delius/reschedule-appointment.spec.ts

```diff
@@ -388,3 +393,35 @@ test('Perform supplier assessment appointment scheduling with conflicting appoin
         'The proposed date and time you selected clashes with another appointment. Please select a different date and time.'
     )
 })
+
+test('Verify Referral Cancellation by Probation Practitioner and its Reflection in Delius', async ({ page }) => {
+    test.slow()
+
+    // Create offender, community event, and requirement
+    const crn = await createOffender(page, { providerName: data.teams.referAndMonitorTestTeam.provider })
+    await createCommunityEvent(page, { crn, allocation: { team: data.teams.referAndMonitorTestTeam } })
+    await createRequirementForEvent(page, { crn, team: data.teams.referAndMonitorTestTeam })
+
+    // Generate a referral and assign it, then create a Supplier Assessment Appointment in R&M
+    const referralRef = await createAndAssignReferral(page, crn)
+    await createSupplierAssessmentAppointment(page, referralRef, addDays(new Date(), 2))
+
+    // Find the correct referral using the Referral Reference & Cancel the Referral
+    await logoutRandM(page)
+    await loginAsPractitioner(page)
+    await cancelReferral(page, referralRef)
+
+    // Verify the referral cancellation should reflect in Delius
+    await loginDelius(page)
+    await navigateToNSIContactDetails(page, crn, true)
+
+    //  Verify that the outcome is set to 'CRS Referral Cancelled'
+    const outcomeSiblingElement = await page.evaluate(() => {
+        const elements = document.querySelectorAll('div.form-group.form-group-sm > div.col-sm-7')
+        const matchingElement = Array.from(elements).find(element =>
+            element.textContent.includes('CRS Referral Cancelled')
+        )
+        return matchingElement?.textContent || null
+    })
+    expect(outcomeSiblingElement).toContain('CRS Referral Cancelled')
```

Comment:
Do you need to run JS on the page here?  Can you not get this with a normal locator?
```suggestion
    await expect(page.locator('div.form-group.form-group-sm > div.col-sm-7')).toContainText('CRS Referral Cancelled')
```
Also I think you're missing the `await`?